config/jobs/kubernetes/sig-k8s-infra/trusted/sig-security-trusted.yaml

```diff
@@ -11,79 +11,21 @@ periodics:
   decorate: true
   extra_refs:
   - org: kubernetes
-    repo: kubernetes
-    base_ref: master
-    path_alias: k8s.io/kubernetes
+    repo: sig-security
+    base_ref: main
+    workdir: true
   spec:
+    serviceAccountName: k8s-snyk-scan
```

Comment:
Do we need to have a service account token at all?

If not, maybe let's set `automountServiceAccountToken: false` instead?

Reply:
we do have some jobs with that, so let's give that a shot :)

https://cs.k8s.io/?q=automountServiceAccountToken%3A%20false&i=nope&files=jobs&excludeFiles=&repos=kubernetes/test-infra

Reply:
I have opened https://github.com/kubernetes/test-infra/issues/33970 as a reminder to remove this service account token once we know the job is running properly from its external script call